demisto_sdk/tests/test_files/content_repo_example/Packs/FeedAzureValid/TestPlaybooks/script-prefixed_automation.yml

```diff
@@ -15,3 +15,5 @@ args:
   description: Amount of seconds to sleep
 scripttarget: 0
 timeout: 1h40m0s
+tests:
+  - No tests
```

Comment:
Do other validations use these test files that you've modified? If so, please check that your additions don't create any issues in those validations.

Reply:
Need to understand better why you're modifying all these test files.

Reply:
Because the enforcement changes things for other tests too where the tests validation is irelevant.